src/chessboard/components/PromotionDialog.tsx

```diff
@@ -37,7 +37,7 @@ export function PromotionDialog() {
       width: "100%",
       height: `${boardWidth / 4}px`,
       top: 0,
-      backgroundColor: "white",
+      backgroundColor: "transparent",
```

Comment:
If you make it's bg transparent, IDK for me its stopps being a modal, and making it transparent is not enough, because the area of div which you've made transparent should not close the dialog, and it can be super confusing for chess players

Reply:
Good call. I'll need to fix that for my implementation. The white background just seemed very out of place compared to the other options.